docs/ContributionGuide.md

```diff
@@ -46,3 +47,7 @@ Following additional environment variables must be provided in order to run the
 |`ADYEN_CLIENT_KEY` | Client-side key that is required to render a Component. See [Adyen documentation](https://docs.adyen.com/development-resources/client-side-authentication#get-your-client-key). | YES |
 
 - Execute `npm run e2e` to run E2e tests.
+
+##### Debugging
+
+E2E tests using [puppeteer](https://github.com/puppeteer/puppeteer) library for testing payment sandbox UI's on the Chrome browser. Refer to puppeteer [debugging tips](https://github.com/puppeteer/puppeteer#debugging-tips) documentation such as turning off headless mode and using nodejs debugger, which might be useful to develop new UI integration or troubleshooting UI flow.
```

Comment:
I use debugger in IntelliJ to do this. Might be also useful if you could mention sth like:
```
If you use IntelliJ Idea, you could also debug using the build-in Mocha test runner since all our e2e tests are written with Mocha.
```

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/873/commits/01597e1209a2e278ce903b1edecd9fda6ecb6524